wcfsetup/install/lang/de.xml

```diff
@@ -1645,6 +1645,7 @@ Wenn Sie unter <em>System -&gt; Optionen -&gt; Allgemein -&gt; E-Mails</em> alle
 		<item name="wcf.acp.user.option.editable.1"><![CDATA[Eigentümer]]></item>
 		<item name="wcf.acp.user.option.editable.2"><![CDATA[Administrator]]></item>
 		<item name="wcf.acp.user.option.editable.3"><![CDATA[Eigentümer und Administrator]]></item>
+		<item name="wcf.acp.user.option.editable.8"><![CDATA[Eigentümer bei Erstanmeldung und Administrator]]></item>
```

Comment:
The problem is, that `EDITABILITY_ALL` is `3` and `6 & 3` is `true`, but `8 & 3` not